web/src/views/DandisetLandingView/DownloadDialog.vue

```diff
@@ -120,14 +120,16 @@ import { computed, ref } from 'vue';
 import { useDandisetStore } from '@/stores/dandiset';
 import CopyText from '@/components/CopyText.vue';
 
-function formatDownloadCommand(identifier: string, version: string): string {
-  if (version === 'draft') {
-    return `dandi download https://dandiarchive.org/dandiset/${identifier}/draft`;
-  }
-  if (!version) {
-    return `dandi download DANDI:${identifier}`;
-  }
-  return `dandi download DANDI:${identifier}/${version}`;
+function downloadCommand(identifier: string, version: string): string {
+  // Use the special 'DANDI:' url prefix if appropriate.
+  const generalUrl = `${window.location.origin}/dandiset/${identifier}`;
+  const dandiUrl = `DANDI:${identifier}`;
+  const url = window.location.origin == 'https://dandiarchive.org' ? dandiUrl : generalUrl;
+
+  // Prepare a url suffix to specify a specific version (or not).
+  const versionPath = version ? `/${version}` : '';
+
+  return `dandi download ${url}${versionPath}`;
```

Comment:
I have simplified the code and I think it still covers all cases. Please let me know if you disagree.

Reply:
This makes sense to me -- thanks Roni.